actions/add-review-labels/index.js

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright IBM Corp. 2020, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+'use strict';
+
+import github from '@actions/github';
+import core from '@actions/core';
+import { App } from 'octokit';
+import util from 'util';
+import decompress from 'decompress';
+
+async function run() {
+  const { context } = github;
+  console.log(context);
+  const appId = core.getInput('APP_ID', {
+    required: true,
+  });
+  const privateKey = core.getInput('APP_PRIVATE_KEY', {
+    required: true,
+  });
+  const app = new App({ appId, privateKey });
+  const octokit = await app.getInstallationOctokit(52238220);
+
+  const { workflow_run, repository, organization } = context.payload;
+  const workflowRunId = workflow_run.id;
+
+  const { data: workflowArtifacts } = await octokit.request(
+    'GET /repos/{owner}/{repo}/actions/runs/{run_id}/artifacts',
+    {
+      owner: organization.login,
+      repo: repository.name,
+      run_id: workflowRunId,
+      headers: {
+        'X-GitHub-Api-Version': '2022-11-28',
+      },
+    }
+  );
+  console.log('workflowArtifacts', workflowArtifacts);
```

Comment:
double checking if we want to be logging stuff out?

Reply:
😱 probably not, those are leftover from when I was testing this. I'll remove it